packages/core/src/routes/admin-user.ts

```diff
@@ -122,10 +125,146 @@ export default function adminUserRoutes<T extends AuthedRouter>(router: T) {
     }
   );
 
+  router.patch(
+    '/users/:userId/password',
+    koaGuard({
+      params: object({ userId: string() }),
+      body: object({ password: string().min(6) }),
+    }),
+    async (ctx, next) => {
+      const {
+        params: { userId },
+        body: { password },
+      } = ctx.guard;
+
+      await findUserById(userId);
+
+      const { passwordEncrypted, passwordEncryptionSalt } = encryptUserPassword(userId, password);
```

Comment:
should we also expose 'passwordEncryptionMethod'?

Reply:
updated